packages/sdk/src/serializationHelpers.ts

```diff
@@ -431,3 +431,10 @@ export const makeSerializeOptional =
 
         return Buffer.concat([encodeBool(true), fun(value)]);
     };
+
+export const makeSerializeList =
```

Comment:
**Suggestion (non-blocking):** Avoid introducing this function and use `serializeList` from this file instead.